recipes/imdclient/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set name = "imdclient" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/imdclient-{{ version }}.tar.gz
+  sha256: 5e18d01b0265d2ea553dd47dbeb0a50e6b8dfb2244c22d9714836267906ddf74
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=61.2
+    - versioningit
+    - pip
+  run:
+    - python >=3.9
+    - mdanalysis >=2.7.0
+
+test:
+  imports:
+    - imdclient
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/Becksteinlab/imdclient
+  summary: Receiver for IMD v2 and v3 data from simulation engines like Gromacs, LAMMPS, and NAMD
+  license: MIT
+  license_file: LICENSE
+  description: |
+    IMD (Interactive Molecular Dynamics) is a protocol for communicating molecular simulation data through a socket.
+    It allows for two-way communication via IMD, a simulation engine sends data to a receiver and the receiver can send forces and certain requests back to the simulation engine.
+    Version numbers in IMD are monotonically increasing integers. 
+    IMDv3, the protocol version described in this document, builds upon IMDv2, which is implemented at the time of writing in NAMD, VMD, GROMACS, and LAMMPS.
+    IMDv2 is itself a modification of the original IMD protocol published in 2001.
+  doc_url: https://imdclient.readthedocs.io/en/latest/protocol_v3.html
+  dev_url: https://github.com/Becksteinlab/imdclient
+
+extra:
+  recipe-maintainers:
+    - hcho38
+    - ljwoods2
+    - amruthesht
```

I am willing to be a maintainer (and the package is hosted in my groups GitHub so I have to ensure that at least one person has long-term access).
```suggestion
    - amruthesht
    - orbeckst
```